packages/azure-openapi-validator/core/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@microsoft.azure/openapi-validator-core",
-  "version": "1.0.0",
+  "version": "1.0.1",
```

Comment:
FYI: If I would want to play this extra-safe, I would change the azure-openapi-validator `package.json` dependency from `"@microsoft.azure/openapi-validator-core": "~1.0.0",` to `"@microsoft.azure/openapi-validator-core": "1.0.0",` and then update it later to `1.0.1`. But I deem the risk acceptable here. If things break, I will deploy core `1.0.2` that reverts to `1.0.0`. There appears to be only one small commit since Nov 2022 to core that needs to be picked up: https://github.com/Azure/azure-openapi-validator/commits/main/packages/azure-openapi-validator/core